operator/pkg/central/extensions/reconcile_pvc.go

```diff
@@ -137,7 +131,8 @@ func (r *reconcilePVCExtensionRun) Execute() error {
 	if r.centralObj.DeletionTimestamp != nil || r.persistence == nil {
 		return r.handleDelete()
 	}
-	if common.ObsoletePVC(r.centralObj.GetAnnotations()) {
```

Comment:
Looks like it would have been disowned if the user placed the annotation there. I thought that was intentional 😅 
Probably no point making this an error at this point.

Reply:
Yes. And I made a fix for this for 4.5.2 and 4.4.5.  FYI: ROX-25836.